features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/wrappers/CanvasWrapper.kt

```diff
@@ -8,19 +8,38 @@ package com.datadog.android.sessionreplay.internal.recorder.wrappers
 
 import android.graphics.Bitmap
 import android.graphics.Canvas
+import com.datadog.android.api.InternalLogger
 
-internal class CanvasWrapper {
+internal class CanvasWrapper(
+    private val logger: InternalLogger = InternalLogger.UNBOUND
```

Comment:
why do you use an unbound logger here ?

Reply:
At the moment the internalLogger is not passed into the base64 structure. The logging change can be made in this PR but it will increase the scope quite a bit, and I preferred to leave it for a later PR where we can fix all the base64 loggers at once